helm/opendj/README.md

```diff
@@ -111,3 +106,56 @@ with an alternate implementation.
 If you are benchmarking on a cloud provider make sure you use an SSD storage class as the directory is very sensitive 
 to disk performance.
 
+## Initializing a DS cluster from a previous backup (experimental)
+
+Backups to gcs are placed in the a directory structure that looks like this:
+
+`gs://backup-bucket/backup-root/{namespace}/{instance}/yy/mm/dd/`
+
+For example:
+
+`gs://forgeops/dj-backup/10m/sandbox/userstore/2018/05/31`
+
+Where 'sandbox' is the namespace name, and 'userstore' is the instance.
+
+The restore script will recurse through the `yy/mm/dd` to find the most recent backup. Within each day folder, there is a full backup and a a number of incrementals. The restore will restore the most recent incremental and full backup for that day.
+
+To recover this backup to a new instance, perform the following procedure:
+
+### Copy the data to a gcs location for the desired namespace
+
+For example, if your namespace is `test` you can do the following:
+
+`gsutil cp -r gs://forgeops/dj-backup/10m/sandbox gs://forgeops/dj-backup/10m/test`
+
+### Prepare your custom.yaml
+
+The custom.yaml for your new cluster will enable gcs restore parameters, and point at your restore bucket from above. For example:
+
+```yaml
+restore:
+  enabled: true
+
+gcs:
+  # Set this to true to enable backups to Google Cloud Storage. You need to create the top level bucket first.
+  restore: true
+  # Restore bucket.
+  restoreBucket: gs://forgeops/dj-backup/10m
+```
+
+Note that the `namespace` and `instance-name` are calculated, are *not* part of the restoreBucket path.
+
+### Install your helm chart
+
+Using the above custom.yaml (note that additional parameters may be required - the sample above shows only
```

Comment:
Sugg: Using the above custom.yaml file, install your helm opendj chart. Note that additional parameters may be required - the sample above shows only the restore features.